app/views/error_with_return_url.njk

```diff
@@ -1,17 +1,17 @@
 {% extends "layout.njk" %}
 
 {% block pageTitle %}
-  An error occurred - GOV.UK Pay
+  {{ i18n.errorViews.error }} - GOV.UK Pay
 {% endblock %}
 
 {% block content %}
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <h3 class="govuk-heading-l">
-      An error occurred:
+      {{ i18n.errorViews.error }}:
     </h3>
     <p class="govuk-body-l lede">
-      <a id="return-url" class="govuk-link" href="{{ returnUrl }}">Return to service</a>
+      <a id="return-url" class="govuk-link" href="{{ returnUrl }}">{{ i18n.authorisation.goBackToService }}</a>
```

Comment:
I’d missed that this was used here: maybe move this to `i18n.commonLinks.goBackToService`?